packages/atomic/src/components/insight/atomic-insight-refine-toggle/atomic-insight-refine-toggle.pcss

```diff
@@ -0,0 +1,14 @@
+atomic-icon-button::part(badge) {
+  background: var(--atomic-primary);
+  color: var(--atomic-on-primary);
+  width: 1rem;
+  display: block;
+  height: 1rem;
+  border-radius: 1rem;
```

Comment:
I prefer using a border radius in percentage, so that if we change the dimensions it stays a circle.